include/qphix/blas_utils.h

```diff
@@ -42,6 +63,69 @@ cnmadd(FT res[2][S], const FT alpha[2], const FT x[2][S], const FT y[2][S])
   }
 }
 
+/**
+  Computes (α τ³ x + β τ¹ x).
+
+  \param[out] res_up Array with complex index and then \c soalen index.
+  \param[out] res_dn Similar, down flavor
+  \param[in] alpha Complex number α
+  \param[in] beta Real number β
+  \param[in] x_up Input up flavor spinor
+  \param[in] x_dn Similar, down flavor
+  */
+template <typename AT, int soalen>
```

Comment:
This is something that I want to adjust in the scope of #41 anyway, so I think we can just either and I will do a global refactoring at some later time.